applications/openshift/kubernetes/master/scheduler/profiling_argument/rule.yml

```diff
@@ -0,0 +1,32 @@
+documentation_complete: true
+
+title: 'Ensure that the --profile argument is set to false'
```

Comment:
Change this to `Ensure that the --profile argument is set` or `Ensure That The --profile Argument Is Configured`